### PR TITLE
Enable Vagrant functional test flow

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -4,26 +4,27 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
-        domain.graphics_type = "spice"
-        domain.memory = 2048
-        domain.video_type = "qxl"
-    end
-
-    dev.vm.provision "ansible" do |ansible|
-        ansible.playbook = "vagrant/rho-dev.yml"
-    end
-  end
+  # config.vm.define "rho-dev" do |dev|
```
Comment (noahl) [inline Vagrantfile]: If we don't need it, let's just remove `rho-dev`.

File: Vagrantfile [hunk 2]
```diff
@@ -4,26 +4,27 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+  #       domain.memory = 2048
+  #       domain.video_type = "qxl"
+  #   end
+  #
+  #   dev.vm.provision "ansible" do |ansible|
+  #       ansible.playbook = "vagrant/rho-dev.yml"
+  #   end
+  # end
 
   # Define a different box to be scanned during tests
```
Comment (noahl) [inline Vagrantfile]: Remove "different," since `rho-dev` no longer exists.

File: library/spit_results.py [hunk 3]
```diff
@@ -34,10 +33,10 @@ def __init__(self, module):
 
     def write_to_csv(self):
         """Output report data to file in csv format"""
-        f_path = self.file_path
+        f_path = os.path.normpath(self.file_path)
         with open(f_path, 'w') as write_file:
             file_size = os.path.getsize(f_path)
-            vals = ast.literal_eval(self.vals)
+            vals = self.vals
```
Comment (noahl) [inline library/spit_results.py:39]: Why are you removing the `ast.literal_eval`? I'm glad to see it gone, but curious why.

**Review by noahl — COMMENTED** [review]
I left a few small comments, but this is awesome and very very important, and I'm excited to get it committed.

**Comment by chambridge** [comment]
Splitting this PR into two pieces.
- Enabling Vagrant support https://github.com/quipucords/rho/pull/36
- Enabling non-default ssh port scan https://github.com/quipucords/rho/issues/37

**Pull request closed without merging**